site/include/header.php

```diff
@@ -1,50 +1,69 @@
 <?php
 
-function PrintHeader(){
-	echo'<div class="HeaderImage">
-		<img src="Curtain.jpg" alt="Curtains"
-		width="100%" height="30%" />
-		<div class="HeaderLine">
-			<h1>
-				St. Louis Theatre
-			</h1>
-		</div>
-	</div>
-	<div class="NavBar">
-		<a href="index.php">
-			<div class="NavButton">
-				<p>
-					Home
-				</p>
-			</div>
-		</a>
-		<a href="News.php">
-			<div class="NavButton">
-				<p>
-					News
-				</p>
-			</div>
-		</a>
-		<a href="Calendar.php">
-			<div class="NavButton">
-				<p>
-					Calendar
-				</p>
+function PrintHeader($SiteTitle, $PageTitle){
+	echo'
```

Comment:
You don't need to change this, but I'd generally recommend using double quotes for the main echo, and single quotes for things like `rel='stylesheet'` below. The only reason it matters is that the PHP tricks I mention in my next comment only work if you use double quotes.